project/Dependencies.scala

```diff
@@ -12,6 +12,8 @@ object Dependencies {
   val AkkaVersionInDocs = "2.6.16"
   val AlpakkaVersionInDocs = "2.0"
   val AlpakkaKafkaVersionInDocs = "2.0"
+  val AkkaGrpcVersionInDocs = "2.1.3"
+  val AkkaPersistenceR2dbcVersionInDocs = Versions.akkaPersistenceR2dbc
```

Comment:
I updated several versions in https://github.com/akka/akka-projection/pull/689
there I also updated pom.xml in the grpc samles.